<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item>
    <widget class="QGroupBox" name="bboxGroup">
     <property name="title">
      <string>Polygon (wkt, espg:4326)</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="polygonInput">
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="polygonBtn">
        <property name="text">
         <string>Use selected</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="drawBtn">
        <property name="text">
         <string>Draw</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tab">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="configTab">
      <attribute name="title">
       <string>Configuration</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="bmGroup">
         <property name="title">
          <string>Basemap authentication</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="1">
           <widget class="QLabel" name="bmPasswordLbl">
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3" colspan="2">
           <widget class="QgsPasswordLineEdit" name="bmPasswordInput"/>
          </item>
          <item row="0" column="3" colspan="2">
           <widget class="QLineEdit" name="bmUsernameInput"/>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="bmUsernameLbl">
            <property name="text">
             <string>Username</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="dtGroup">
         <property name="title">
          <string>Data authentication</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="dtApikeyLbl">
            <property name="text">
             <string>API key</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QgsPasswordLineEdit" name="dtApikeyInput"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="searchGroup">
         <property name="title">
          <string>Search results</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="QLabel" name="maxResultsLbl">
            <property name="text">
             <string>Max results</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="maxResultsInput">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>500</number>
            </property>
            <property name="value">
             <number>500</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="maxResultsSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="configTabSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="bmTab">
      <attribute name="title">
       <string>BaseMap</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="bmDateGroup">
         <property name="title">
          <string>Inserted Date</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QgsDateTimeEdit" name="bmFromInput">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="allowNull" stdset="0">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QgsDateTimeEdit" name="bmToInput">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="accelerated">
             <bool>false</bool>
            </property>
            <property name="allowNull" stdset="0">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="bmFromCheck">
            <property name="text">
             <string>From</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="bmToCheck">
            <property name="text">
             <string>To</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="bmTabSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="dtTab">
      <attribute name="title">
       <string>Data</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="dtSensorGroup">
         <property name="title">
          <string>Sensor / Constellation</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QCheckBox" name="dtSpotCheck">
            <property name="text">
             <string>Spot 6/7 (SPOT)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="dtPleiadesCheck">
            <property name="text">
             <string>Pleiades (PHR)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="dtPleiadesNeoCheck">
            <property name="text">
             <string>Pleiades Neo (PNEO)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="dtDateGroup">
         <property name="title">
          <string>Acquisition Date</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="2">
           <widget class="QCheckBox" name="dtToCheck">
            <property name="text">
             <string>To</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="dtFromCheck">
            <property name="text">
             <string>From</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <property name="tristate">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QgsDateTimeEdit" name="dtToInput">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
            <property name="allowNull" stdset="0">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QgsDateTimeEdit" name="dtFromInput">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::UpDownArrows</enum>
            </property>
            <property name="showGroupSeparator" stdset="0">
             <bool>false</bool>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
            <property name="allowNull" stdset="0">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="dtAngleGroup">
         <property name="title">
          <string>Incidence Angle</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="2">
           <spacer name="dtAngleMinSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="dtAngleMinCheck">
            <property name="text">
             <string>Min</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <spacer name="dtAngleMaxSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="3">
           <widget class="QCheckBox" name="dtAngleMaxCheck">
            <property name="text">
             <string>Max</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QSpinBox" name="dtAngleMaxInput">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="wrapping">
             <bool>false</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="suffix">
             <string>°</string>
            </property>
            <property name="maximum">
             <number>30</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="dtAngleMinInput">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="suffix">
             <string>°</string>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="maximum">
             <number>30</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="dtCoverGroup">
         <property name="title">
          <string>Max Cover</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="2">
           <spacer name="dtCloudSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="dtCloudCheck">
            <property name="text">
             <string>Cloud</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <spacer name="dtSnowSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="3">
           <widget class="QCheckBox" name="dtSnowCheck">
            <property name="text">
             <string>Snow</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QSpinBox" name="dtSnowInput">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="suffix">
             <string>%</string>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="dtCloudInput">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="suffix">
             <string>%</string>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>5</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="dtWorkspaceGroup">
         <property name="title">
          <string>Workspace (at least one)</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <widget class="QCheckBox" name="dtPublicCheck">
            <property name="text">
             <string>Public</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="dtPrivateCheck">
            <property name="text">
             <string>Private</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="dtTabSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="errorLbl">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>204</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>204</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>190</red>
           <green>190</green>
           <blue>190</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="dialogBtnLayout">
     <item alignment="Qt::AlignLeft">
      <widget class="QDialogButtonBox" name="dialogBtnBox">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
       <property name="centerButtons">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="dialogBtnSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="searchBtn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>search.png</normaloff>search.png</iconset>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDateTimeEdit</class>
   <extends>QDateTimeEdit</extends>
   <header>qgsdatetimeedit.h</header>
  </customwidget>
  <customwidget>
   <class>QgsPasswordLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qgspasswordlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>polygonInput</tabstop>
  <tabstop>polygonBtn</tabstop>
  <tabstop>tab</tabstop>
  <tabstop>bmUsernameInput</tabstop>
  <tabstop>bmPasswordInput</tabstop>
  <tabstop>dtApikeyInput</tabstop>
  <tabstop>bmFromCheck</tabstop>
  <tabstop>bmFromInput</tabstop>
  <tabstop>bmToCheck</tabstop>
  <tabstop>bmToInput</tabstop>
  <tabstop>dtSpotCheck</tabstop>
  <tabstop>dtPleiadesCheck</tabstop>
  <tabstop>dtFromCheck</tabstop>
  <tabstop>dtFromInput</tabstop>
  <tabstop>dtToCheck</tabstop>
  <tabstop>dtToInput</tabstop>
  <tabstop>dtAngleMinCheck</tabstop>
  <tabstop>dtAngleMinInput</tabstop>
  <tabstop>dtAngleMaxCheck</tabstop>
  <tabstop>dtAngleMaxInput</tabstop>
  <tabstop>dtCloudCheck</tabstop>
  <tabstop>dtCloudInput</tabstop>
  <tabstop>dtSnowCheck</tabstop>
  <tabstop>dtSnowInput</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>dialogBtnBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>581</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>490</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialogBtnBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>581</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>489</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dtAngleMinCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>dtAngleMinInput</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dtAngleMaxCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>dtAngleMaxInput</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bmFromCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>bmFromInput</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bmToCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>bmToInput</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dtFromCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>dtFromInput</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dtToCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>dtToInput</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dtCloudCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>dtCloudInput</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>429</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dtSnowCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>dtSnowInput</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
